backend/src/controllers/v2/secretController.ts

```diff
@@ -61,15 +86,15 @@ export const batchCreateSecrets = async (req: Request, res: Response) => {
       secretCommentTag: rawSecret.secretCommentTag,
       secretCommentHash: rawSecret.secretCommentHash,
       workspace: new Types.ObjectId(workspaceId),
-      environment: environmentName,
+      environment,
       type: rawSecret.type,
       user: new Types.ObjectId(req.user._id)
     }
 
     sanitizedSecretesToCreate.push(safeUpdateFields)
   })
 
-  const [bulkCreateError, newlyCreatedSecrets] = await to(Secret.insertMany(sanitizedSecretesToCreate).then())
+  const [bulkCreateError, secrets] = await to(Secret.insertMany(sanitizedSecretesToCreate).then())
```

Comment:
Is there a Typo? Isn't the correct form is "sanitizedSecretsToCreate"

Reply:
I think it might be, but this controller is not user anymore. We will deprecate it soon, but for now the correct version is in `secretsContoller.ts`